src/game/models/base_card.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from game.models.enums.card_type import CardType
+
+
+class BaseCard(models.Model):
+    id = models.SlugField(max_length=30, primary_key=True)
+    name = models.SlugField(max_length=30)
+    type = models.CharField(choices=CardType.choices,
+                            max_length=max([len(card_name) for card_name in CardType.values]),
+                            default=None, null=False, blank=False)
```

Comment:
Hoekom is die default `None` maar jy se `null=False`? Essentially se jy by default moet jy null wees, maar jy mag nie null wees nie... -_-

Reply:
Ook. jy kan die `type` field 'n slugField maak